### PR TITLE
Feature/canvas size filter

### PR DESCRIPTION
**Pull request by rohankul**

### CODE REVIEW

File: src/app/shared/creative-editor/creative-editor.component.ts [hunk 1]
```diff
@@ -116,8 +154,8 @@ export class CreativeEditorComponent implements OnInit,AfterViewInit,OnDestroy
     this.filtered.push(block6);
     this.filtered.push(block3);
 
-    this.canvasHeight = 250;
-    
+    //this.canvasHeight = 250;
```
Comment (amolra) [inline src/app/shared/creative-editor/creative-editor.component.ts]: Please remove this commented code

File: src/app/shared/creative-editor/creative-editor.component.ts [hunk 2]
```diff
@@ -249,7 +289,25 @@ export class CreativeEditorComponent implements OnInit,AfterViewInit,OnDestroy
...
+            width: STYLE.LANDSCAPE.SIZE.WIDTH, // this value will be used on canvas width
+            height: STYLE.LANDSCAPE.SIZE.HEIGHT
+        },
+        {
+            name: STYLE.PORTRAIT.NAME,
+            width: STYLE.PORTRAIT.SIZE.WIDTH, // this value will be used on canvas width
+            height: STYLE.PORTRAIT.SIZE.HEIGHT
+        }]
+      },
+      panels: { defaults: []
```
Comment (amolra) [inline src/app/shared/creative-editor/creative-editor.component.ts]: Please align it properly

File: src/assets/styles/scss/_gjs_canvas.scss [hunk 3]
```diff
@@ -192,7 +192,8 @@ $guide_pad: 5px;
 }
 
 .#{$cv-prefix}canvas {
-  background-color: rgba(0, 0, 0, 0.15);
+  /*background-color: rgba(0, 0, 0, 0.15);*/
```
Comment (amolra) [inline src/assets/styles/scss/_gjs_canvas.scss:195]: Remove this commented code

**Review by ajaypa-cybage — APPROVED** [review]

File: src/app/shared/creative-editor/creative-editor.component.css [hunk 4]
```diff
@@ -118,12 +118,20 @@
   letter-spacing: normal;
   color: #e6eaed;
 }
-.editor.basic, .editor.landscape {
-  margin-top: 25%;
+
+.editor {
+  /* width: 300px;
+  height: 250px; */
```
Comment (ajaypa-cybage) [inline src/app/shared/creative-editor/creative-editor.component.css]: Remove commented code.

File: src/app/shared/creative-editor/creative-editor.component.css [hunk 5]
```diff
@@ -118,12 +118,20 @@
...
+  margin: 150px 0 0 350px;
+  /* padding: 5px 8px 16px 22px; */
```
Comment (ajaypa-cybage) [inline src/app/shared/creative-editor/creative-editor.component.css]: Please remove any commented code.

File: src/app/shared/creative-editor/creative-editor.component.html [hunk 6]
```diff
@@ -1,39 +1,45 @@
...
-                      </div>
-                </div>
-            <!-- </div> -->
-            <div class="card block-manager">
-                <p class="block-text">AD ELEMENTS</p>
-                <button class="btn ad-element-button"><i class="fa fa-plus" style="margin-right:20%;margin-left:-30%;"></i>Add Element</button>
-                <div #customid class="block-manager-gjs"></div>
-                <!-- <hr style="border:1px solid #626875;width:114%;">  -->
-                <!-- <button class="btn ad-button"><i class="fa fa-plus" style="margin-right:20%;margin-left:-45%;"></i>Square</button>
+<div class="row" style="margin-top: 61px;">
```
Comment (ajaypa-cybage) [inline src/app/shared/creative-editor/creative-editor.component.html]: Please avoid any inline style to be added.

File: src/app/shared/creative-editor/creative-editor.component.html [hunk 7]
```diff
@@ -1,39 +1,45 @@
...
+      <!-- </div> -->
+    </div>
+    <div class="block-manager">
+      <p class="block-text">AD ELEMENTS</p>
+      <button class="ad-element-button">
+        <i class="fa fa-plus" style="margin-right: 20%;margin-left: -20%;"></i><span class="ad-element-btn-text">Add Element</span>
+      </button>
+      <div #customid class="block-manager-gjs"></div>
+      <!-- <hr style="border:1px solid #626875;width:114%;">  -->
+      <!-- <button class="btn ad-button"><i class="fa fa-plus" style="margin-right:20%;margin-left:-45%;"></i>Square</button>
```
Comment (ajaypa-cybage) [inline src/app/shared/creative-editor/creative-editor.component.html]: Please remove commented code.

**Review by ajaypa-cybage — APPROVED** [review]

**Pull request merged**